dns-hosting.md

```diff
@@ -0,0 +1,20 @@
+---
+title: DNS hosting
+layout: recommendation
+expires: 2018-01-01
+---
+To make services available on the internet they need to have DNS records to let people know where they are.
+service.gov.uk domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by service teams.
+
+
+## Principles
+
+- DNS entry configuration should come from a version-controlled repository unless autogenerated
+- consider nameserver availability as part of your overall service availability
```

Comment:
haha, just realised [I wrote this text in the first place](https://github.com/alphagov/gds-tech/pull/27#discussion_r121356344)